OutOfSchool/OutOfSchool.WebApi/Startup.cs

```diff
@@ -226,7 +229,9 @@ public static void AddApplicationServices(this WebApplicationBuilder builder)
         services.AddTransient<IParentRepository, ParentRepository>();
         services.AddTransient<IProviderRepository, ProviderRepository>();
         services.AddTransient<IRatingRepository, RatingRepository>();
```

Comment:
and probably this as now it will have default implementation

Reply:
Removed.